website/addons/googledrive/utils.py

```diff
@@ -27,6 +28,7 @@ def to_hgrid(item, node, path):
         'kind': 'folder',
         'name': safe_name,
         'addon': 'googledrive',
+        'provider': 'googledrive',  # TODO: remove duplication -- APIv2
```

Comment:
>Is `to_hgrid` sending things to the v2 api serializer? That seems less than ideal. You could just use the source field on the v2 serializer to tell it which field to use. Or am I misunderstanding?

Yes, kind of. There's some amount of normalization required between receiving folder info from any third party, most addons have some helper to do this, some do it in the same function that makes the `GET` request. Be not alarmed, `rubeus` isn't actually being used.

`source=` will probably work.